frontend/app/src/store/history/actions.ts

```diff
@@ -219,7 +219,25 @@ export const actions: ActionTree<HistoryState, RotkehlchenState> = {
       onlyCache: false
     };
 
-    await api.history.trades(params);
+    const { awaitTask } = useTasks();
```

Comment:
We not trigger the action for fetching with only cache false? Instead of copying